### PR TITLE
EOS-15937: Kafka server installation

### PR DESCRIPTION
**Pull request by SomwanshiAnjali**
Signed-off-by: Anjali Somwanshi <anjali.somwanshi@seagate.com>

### CODE REVIEW

**Review by andkononykhin2 — APPROVED** [review]

File: srv/components/misc_pkgs/kafka/install.sls [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+
+{% import_yaml 'components/defaults.yaml' as defaults %}
+
+{% set kafka_version = pillar['commons']['version']['kafka'] %}
+
+Install Java:
+  pkg.installed:
+    - pkgs:
+      - java-1.8.0-openjdk
+      - java-1.8.0-openjdk-devel
```
Comment (ypise) [inline srv/components/misc_pkgs/kafka/install.sls:26]: Not sure if we need devel. Usually it's avoided on Prod systems (comment the line).
Also, let's try to align this with S3Server which uses: `java-1.8.0-openjdk-headless`

File: srv/components/misc_pkgs/kafka/start.sls [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+# You should have received a copy of the GNU Affero General Public License
+# along with this program. If not, see <https://www.gnu.org/licenses/>.
+# For any questions about this software or licensing,
+# please email opensource@seagate.com or cortx-questions@seagate.com.
+#
+
+{% set kafka_version = pillar['commons']['version']['kafka'] %}
+
+Start zoopkeper:
+  cmd.run:
```
Comment (ypise) [inline srv/components/misc_pkgs/kafka/start.sls:21]: Could we check if there's already a running instance and execute only if not, using unless?

**Review by ypise — CHANGES_REQUESTED** [review]

File: srv/components/misc_pkgs/kafka/install.sls (hunk 1 above)
Comment (SomwanshiAnjali) [reply srv/components/misc_pkgs/kafka/install.sls]: Addressed

File: srv/components/misc_pkgs/kafka/start.sls (hunk 2 above)
Comment (SomwanshiAnjali) [reply srv/components/misc_pkgs/kafka/start.sls:21]: Addressed

**Review by ypise — APPROVED** [review]

**Pull request merged**